USAGE_android.md

```diff
@@ -7,29 +7,136 @@
 [1]: https://i.creativecommons.org/l/by-nd/4.0/88x31.png "Creative Commons License"
 [2]: https://creativecommons.org/licenses/by-nd/4.0/
 
-Copyright &copy; 2018-2021 LunarG, Inc.
+Copyright &copy; 2018-2023 LunarG, Inc.
 
-# GFXReconstruct API Capture and Replay
+# GFXReconstruct API Capture and Replay - Android
 
-## **Android Version**
+***This document describes the GFXReconstruct software for capturing and
+replaying Vulkan API calls on Android devices.***
+
+If you are looking for capturing/replaying on a different platform, please refer
+to one of these other documents:
+ * [GfxReconstruct for Desktop Vulkan](./USAGE_desktop_Vulkan.md)
+ * [GfxReconstruct for Desktop D3D12](./USAGE_desktop_D3D12.md)
 
-This document describes the GFXReconstruct software for capturing and replaying
-Vulkan API calls on Android devices.
 
 ## Index
 
-1. [Capturing API Calls](#capturing-api-calls)
+1. [Behavior on Android](#behavior-on-android)
+    1. [Framework Updates](#framework-updates)
+    2. [Platform Differences](#platform-differences)
+    3. [Android Writable Locations](#android-writable-locations)
+2. [Capturing API Calls](#capturing-api-calls)
     1. [Before Use](#before-use)
+    2. [To Root or Not To Root](#to-root-or-not-to-root)
     2. [Enabling the Layer with ADB](#enabling-the-layer-with-adb)
     3. [Capture Options](#capture-options)
     4. [Capture Files](#capture-files)
-2. [Replaying API Calls](#replaying-api-calls)
+    5. [Limitations of Capture On Android](#limitations-of-capture-on-android)
+    6. [Known Capture Issues](#known-capture-issues)
+    7. [Troubleshooting Capturing of Applications](#troubleshooting-capturing-of-applications)
+3. [Replaying API Calls](#replaying-api-calls)
     1. [Launch Script](#launch-script)
     2. [Install APK Command](#install-apk-command)
     3. [Replay Command](#replay-command)
-    2. [Touch Controls](#touch-controls)
-    2. [Key Controls](#key-controls)
-    3. [Virtual Swapchain](#virtual-swapchain)
+    4. [Touch Controls](#touch-controls)
+    5. [Key Controls](#key-controls)
+    6. [Limitations of Replay On Android](#limitations-of-replay-on-android)
+    7. [Troubleshooting Replay of Applications](#troubleshooting-replay-of-applications)
+
+## Behavior on Android
+
+The purpose of this section is to describe some of the software changes made to
+the GFXReconstruct software to add Android support.
+This section will not provide a comprehensive list of changes, but will instead
+highlight some of the primary adjustments required to adapt the GFXReconstruct
+software to the Android ecosystem.
+
+### Framework Updates
```

Comment:
I removed them for now.  I do believe this info is useful in general because it explains behavior of the capture tool that is unique to Android and the reasoning behind it.  But I'm ok with it not being in the USAGE doc.